qemu/tests/hv_type.py

```diff
@@ -0,0 +1,27 @@
+import logging
+
+from virttest import error_context
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Check the Hyper-V type
+
+    1) boot the guest with all flags
+    2) check Hyper-V type in guest
+
+    param test: the test object
+    param params: the test params
+    param env: the test env object
+    """
+    timeout = params.get("timeout", 360)
+
+    error_context.context("Boot the guest with all flags", logging.info)
+    vm = env.get_vm(params["main_vm"])
+    vm.verify_alive()
+    session = vm.wait_for_login(timeout=timeout)
+    type = session.cmd("virt-what")
```

Comment:
```suggestion
    hv_type = session.cmd("virt-what")
```
since `type` is built-in function in python.